README.md

```diff
@@ -1,55 +1,21 @@
 <!-- Badges go here (see [shields.io](https://shields.io/), for examples). -->
 
-# Data Engineering Generator
+# Data Generator
 
-This monorepo contains the different technological arms of Scott Logic's Data Generator project, which allows users to automatically analyse their real data to create a profile, and then use that profile to generate realistic dummy data.
+### Welcome to the _Scott Logics_ data generator.
+The generation of representative test and simulation data is challenging and time-consuming. The data-generator allows you to quickly create data, based on a schema, using a variety of generation modes for the purpose of testing and validation.
 
-There are 3 discrete primary elements:
-
-| Product | Description | Status | Notes |
-| ---- | ---- | ---- | ---- |
-| Profiler | Analyse a data source and generate a profile | Stable - Pre-alpha | Currently being developed |
-| Profile | A representation of the data schema and rules which can be used to generate data | Stable - alpha | |
-| Generator | A tool for generating data from a given profile | Stable - alpha | Supports data generation and generation of data that has been [delibrately violated](./generator/docs/DeliberateViolation.md). |
-
-See [overview](./docs/Profiles.md) and [schema](./docs/Schema.md) for details about profiles and how they work.
-
-## Installation
+You can use this data generator to commission and maintain your test systems with data that conforms to your live environments without copying data or breaching GDPR.
 
-This product requires **Java version 1.8** to be installed. Later versions are not supported at present.
-* Clone the repository
-* Follow the steps in the [profiler setup instructions](profiler/README.md). The following need to be actioned for the data generator product, the remaining steps need to be actioned to be able to use the profiler.
-
-  | Generator | Profiler |
-  | ---- | ---- |
-  | JDK | JDK |
-  |  | Scala |
-  |  | Spark |
-  |  | Hadoop |
-  |  | Windows shims |
-  | Cucumber | Cucumber | 
-  | IntelliJ/Eclipse IDE | IntelliJ/Eclipse IDE |
-* Build the solution using your preferred method - the project is fully compatible with IntelliJ and Eclipse
-* Run the data generator or profiler - see the examples below.
+The data generator has been written in Java to allow cross platform compatibility, allowing it to work on Microsoft Windows, Apple Mac and Linux equally.
 
-The product is confirmed compatible with Microsoft Windows, and should be compatible with all other operating systems, but this has not been confirmed to date.
+## Getting started
+As the data generator is at an _beta_ stage, there are no published releases. To be able to use this tool, please build the tool then you'll be able to use it as with the examples below.
```

Comment:
> please **build the tool** then

can you make that a link to the instructions on how to build it? we want to make it as easy as possible for first-timers!

Reply:
Updated to link to the development setup instructions, i guess we need to prioritise getting a release built so we can remove this statement.